app/src/main/res/menu/home_menu.xml

```diff
@@ -50,6 +50,10 @@
       android:title="@string/menu_jump"
       app:showAsAction="never" />
   <item
+      android:id="@+id/translations"
+      android:title="Translations"
```

Comment:
hmm, it appears Spinners don't allow for setting this on the later APIs. "Spinner overrides getViewTypeCount() on the Adapter associated with this view. Calling getItemViewType(int) on the object returned from getAdapter() will always return 0. Calling getViewTypeCount() will always return 1. On API LOLLIPOP and above, attempting to set an adapter with more than one view type will throw an IllegalArgumentException." -https://developer.android.com/reference/android/widget/Spinner.html#setAdapter(android.widget.SpinnerAdapter) also https://code.google.com/p/android/issues/detail?id=17128. It looks like there can be a workaround using Tags however which I can look at